apps/cowswap-frontend/src/modules/orders/pure/OrderSummary/summaryTemplates.tsx

```diff
@@ -1,23 +1,19 @@
-import { ReactElement } from 'react'
+import { ReactElement, ReactNode } from 'react'
 
 export interface OrderSummaryTemplateProps {
   inputAmount: ReactElement
   outputAmount: ReactElement
 }
 
-// TODO: Add proper return type annotation
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-export function SellForAtLeastTemplate({ inputAmount, outputAmount }: OrderSummaryTemplateProps) {
+export function SellForAtLeastTemplate({ inputAmount, outputAmount }: OrderSummaryTemplateProps): ReactNode {
   return (
     <>
       Sell {inputAmount} for at least {outputAmount}
```

Comment:
Thinking for the bridge toast, if we should instead say

>Bridge {inputAmount} for at least {outputAmount}

WDYT @elena-zh ?

Reply:
Makes sense, thanks!
Fixed: https://github.com/cowprotocol/cowswap/pull/5977/commits/1d1144e506e01e20a27080160bca527d3ce3d61c